salt/utils/reactor.py

```diff
@@ -346,7 +346,7 @@ def caller(self, fun, *args, **kwargs):
         Wrap Caller to enable executing :ref:`caller modules <all-salt.caller>`
         '''
         log.debug("in caller with fun {0} args {1} kwargs {2}".format(fun, args, kwargs))
-        args = kwargs.get('args', [])
```

Comment:
I remember seeing it in kwargs at some point.

It might not be there anymore, but i don't see the point in removing it if it could possibly break for some way that this works.